zbus/tests/basic.rs

```diff
@@ -322,3 +323,53 @@ async fn test_freedesktop_api() -> Result<()> {
 
     Ok(())
 }
+
+#[cfg(unix)]
+#[tokio::test]
+#[timeout(15000)]
+#[instrument]
+async fn test_freedesktop_credentials() -> Result<()> {
+    use nix::unistd::{Gid, Uid};
+    use std::os::fd::AsRawFd;
```

Comment:
Nitpick: better use `AsFd` instead. I'm sure `AsRawFd` use will trigger a warning from clippy in the future.

Reply:
Ah nm. `BorrowedFd` doesn't implement Display.